<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>freeHand</class>
 <widget class="QWidget" name="freeHand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cut_freeHand_press">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>FreeHandDrawing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refreshBackground">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Background</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>screenshot geometry</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>57</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>w</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>h</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSpinBox" name="spinBox_screenShot_x"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_screenShot_y"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_screenShot_w"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_screenShot_h"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>velocity</string>
      </property>
      <property name="buddy">
       <cstring>spinBox_velocity</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_velocity"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>120</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>delayFactor</string>
      </property>
      <property name="buddy">
       <cstring>spinBox_velocity</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_delayFactor">
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="maximum">
       <double>2.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
